examples/xor-mlp/src/xor_mlp/core.clj

```diff
@@ -0,0 +1,25 @@
+(ns xor-mlp.core
+  (:require [cortex.experiment.train :as train]
+            [cortex.nn.execute :as execute]
+            [cortex.nn.layers :as layers]
+            [cortex.nn.network :as network]))
+
+;; Input and test data (the xor function)
+(def xor
```

Comment:
This is a really tiny thing, but the symbol `xor` at first blush to me I would expect to be a function that performs and `xor`.  To me, it might be more clear to rename this symbol `xor-dataset` (since we conceptually emphasize that a dataset is a sequence of maps, and that's precisely what we have here) :)